src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
 // write code here
+const listItems = document.querySelectorAll('li');
+
+listItems.forEach((item) => {
+  if (item.querySelector('ul')) {
+    const header = item.firstChild;
+
+    if (header.nodeType === 3) {
+      const pTag = document.createElement('span');
+
+      pTag.textContent = header.textContent;
+      item.insertBefore(pTag, item.firstChild);
+      item.removeChild(header);
+
+      pTag.addEventListener('click', (e) => {
+        const nextUl = pTag.nextElementSibling;
+
+        if (nextUl.style.display === 'none') {
```

Comment:
The initial display state of the `nextUl` element is not set. This could lead to inconsistent behavior if the initial state is not 'none'. Consider setting the initial display state of all sublists to 'none' when the page loads.